TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -5,15 +5,23 @@ import PromiseKit
     var tradeItSessionProvider: TradeItSessionProvider
     public var linkedBrokers: [TradeItLinkedBroker] = []
     
-    init(apiKey: String, environment: TradeitEmsEnvironments) {
+    public init(apiKey: String, environment: TradeitEmsEnvironments) {
         // TODO: TradeItConnector initializer returns optional - we should not force unwrap
-        tradeItConnector = TradeItConnector(apiKey: apiKey, environment: environment, version: TradeItEmsApiVersion_2)!
-        tradeItSessionProvider = TradeItSessionProvider()
+        self.tradeItConnector = TradeItConnector(apiKey: apiKey, environment: environment, version: TradeItEmsApiVersion_2)!
+        self.tradeItSessionProvider = TradeItSessionProvider()
 
         super.init()
         self.loadLinkedBrokersFromKeychain()
     }
     
+    init(tradeItConnector: TradeItConnector) {
```

Comment:
Don't know if there is another way, but we need to inject a FakeTradeItConnector in our unit test